### ISSUE TITLE
colorblind friendly grouping stage interface

### ISSUE THREAD

**Issue by vanderhoop** [stride-nyc/remote_retro#545]
As a colorblind user in the grouping stage,
I want to be able to discern distinct groupings without relying on color distinction,
So that I can participate in the stage without asking teammates for assistance.

__By Way Of Explanation:__ our grouping stage (currently) relies leverages colors to differentiate between groups, color-blind users will have a hard time distinguishing between groups

__Acceptance Criteria:__

- [x] there is a "low vision" icon button in the grouping stage's lower whose `title` attribute reads "Toggle Color-Blind Friendly Grouping"
- [x] upon clicking the low vision button "Toggle Color-Blind Friendly Grouping", colored box-shadows are removed from any groupings, and instead, every card in a given group has a mini floating black label in the upper right hand corner, identifying its group id
  - __Note:__ due to the positioning of this label, we may have to increase padding so that it doesn't overlap the idea text. That's okay.
- [x] re-clicking the "Toggle Color-Blind Friendly Grouping" toggles the coloration back

__Helpful Links:__

- [Semantic UI Icons](https://semantic-ui.com/elements/icon.html)
- [Semantic UI Floating Labels](https://semantic-ui.com/elements/label.html)

__Example:__

![image](https://user-images.githubusercontent.com/4386645/63460086-10652e80-c413-11e9-91ee-9c0515a7d2f2.png)

**Comment by vanderhoop** [comment]
implemented!
